roles/adoption_osp_deploy/tasks/gather_stack_nodes.yml

```diff
@@ -14,6 +14,10 @@
       }}
 
 - name: Gather other facts for stack {{ _overcloud_name }}"
+  when:
+    - "'hostname_groups_map' in cifmw_adoption_osp_deploy_scenario"
+    - "'roles_groups_map' in cifmw_adoption_osp_deploy_scenario"
+    - _network_data_file is defined and _network_data_file
```

Comment:
Will the job actually be able to run if we don't execute the set_fact below? Or will we just defer the failure to a later point?

Reply:
checking now by removing condition